plutus-benchmark/script-contexts/test/checkScriptContextEqualityTerm-20.eval.golden

```diff
@@ -1,2 +1,2 @@
-({cpu: 530200251
-| mem: 2147246})
\ No newline at end of file
+({cpu: 310936611
+| mem: 1245566})
```

Comment:
A bit worse than last time (it was `243063611`, but with a buggy version, so who knows if it even was correct), but still good.